x-pack/plugins/session_view/public/components/process_tree/hooks.ts

```diff
@@ -42,6 +42,7 @@ export class ProcessImpl implements Process {
     this.searchMatched = null;
   }
 
+
```

Comment:
Not sure if eslint complains about the extra newline, but you may want to run: node scripts/eslint --fix  x-pack/plugins/session_view to fix any whitespace issues.